server/app/views/admin/migration/AdminImportView.java

```diff
@@ -32,14 +32,15 @@
  */
 public class AdminImportView extends BaseHtmlView {
   /**
-   * Play Framework defaults to 100KB memory limit per request. We set this to 256KB in
-   * application.conf to allow large programs to be imported. More info:
+   * Play Framework defaults to 100KB memory limit per request which is too small for many programs.
+   * We set this to 512KB in {@link parsers.ProgramMigrationParser} to allow large programs to be
```

Comment:
```suggestion
   * We use a custom body parser {@link parsers.ProgramMigrationParser} to allow large programs to be
```
The size in KB differs from what the parser is configured for. I'd say leave the actual value out of this comment so it'll never get out of sync.